scripts/render_nlp_tags.py

```diff
@@ -0,0 +1,77 @@
+"""Keras Generate Presets Table Implementation
```

Comment:
I don't think we need quite as long a file level docstring here. Maybe just

`"""Custom rendering code for the /api/keras_nlp/models page."""`